api/src/paths/taxonomy/species/list.ts

```diff
@@ -77,7 +77,7 @@ export function getSpeciesFromIds(): RequestHandler {
     try {
       const taxonomyService = new TaxonomyService();
       const response = await taxonomyService.getSpeciesFromIds(ids as string[]);
-
+      res.setHeader('Cache-Control', 'max-age=604800')
```

Comment:
A comment explaining how long this is would be nice (e.g. `// 60 minutes`)

Reply:
+1. 

To add additional context, could say something like: `// Overwrite default cache-control header, allow caching up to 60 minutes`

Reply:
Done